rbd/rbd.go

```diff
@@ -808,6 +808,35 @@ func (image *Image) WriteAt(data []byte, off int64) (n int, err error) {
 	return ret, err
 }
 
+// WriteSame repeats writing data from startpoint off until n bytes have been written.
```

Comment:
"startpoint" -> "the starting point" ?
"off" -> "ofs" ?

Reply:
Done!